src/state_db.rs

```diff
@@ -162,12 +160,13 @@ impl StateDB {
         state_root: Vec<u8>,
         cb: Root<JsFunction>,
     ) -> Result<(), mpsc::SendError<options::DbMessage>> {
-        let key_length = self.options.key_length;
+        let key_length = self.options.key_length();
         self.common.send(move |conn, channel| {
             let result = StateDB::get_revert_result(conn, height, &state_root, key_length);
             if result.is_ok() {
                 let value = (**result.as_ref().unwrap().lock().unwrap()).clone();
-                let state_info = CurrentState::new(&value, height.sub(1).into());
+                let version = height.sub(1).into();
```

Comment:
probably this input name also could be version?

Reply:
we use `height` for the `revert` function if the `revert` should take `version` so it would be correct to change it to `version` if not I think it is better to have height and convert it to version like current implementation.

Reply:
i think to be slightly agnostic from the blockchain, we should change `block height` => `version`. So in the revert function also we can change it to version

Reply:
Done.